src/adapters/lexicons/adapter.js

```diff
@@ -7,23 +7,123 @@ import DefaultConfig from '@/adapters/lexicons/config.json'
 let cachedDefinitions = new Map()
 
 class AlpheiosLexiconsAdapter extends BaseAdapter {
+  /**
+  * Lexicons adapter uploads config data, defines default options and inits data
+  * @param {Object} config - properties with higher priority
+  */
   constructor (config = {}) {
     super()
     this.config = this.uploadConfig(config, DefaultConfig)
     this.options = { timeout: this.config.timeout ? this.config.timeout : 0 }
-    this.data = null
   }
 
+  /**
+  * This method retrieves short definitions for given homonym
+  * @param {Homonym} homonym - homonym for retrieving definitions
+  * @param {Object} options - options
+  * @return {Boolean} - result of fetching
```

Comment:
It says that it returns Boolean here, but it does not return anything. Probably we should either fix the docs or a function, whichever is correct

Reply:
Fixed